extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/AuthenticationProviderService.java

```diff
@@ -307,16 +312,39 @@ public LDAPAuthenticatedUser authenticateUser(Credentials credentials)
 
     }
 
+    /**
+     * Returns parameter current ldap domain token generated from user credentials
+     * @param credentials
+     *     The credentials used for authentication.
```

Comment:
To comply with JavaDoc standards and the code style we use for comments, this should be formatted as:

```
   /**
     * Returns parameter current ldap domain token generated from user credentials.
     *
     * @param credentials
     *     The credentials used for authentication.
```

That said, the phrase "Returns parameter current ldap domain token generated from user credentials" does not make sense as written. It should probably be rephrased.